notebooks/feature_engineering/raw/ex1.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Exercise 1: Building a Baseline Model\n",
+    "# Introduction\n",
```

Comment:
The notebook header (outside the text) already has the title, so we call this section the Introduction (across learn notebooks in general)